scripts/starknet_ibc_core.ts

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env -S deno run
+
+import { Contract, RpcProvider } from "npm:starknet";
+import { QueryClient, setupIbcExtension } from "npm:@cosmjs/stargate";
+import { Tendermint34Client } from "npm:@cosmjs/tendermint-rpc";
+
+const OSMOSIS_RPC_ENDPOINT = "https://rpc.testnet.osmosis.zone";
```

Comment:
Should these come from env?

Reply:
I wanted to leave this as it is, because these two scripts are solely for educational purpose or manual debugging of testnet. They are not meant for automated steps.

Note, I can also generate the output from `cargo run -p hermes-starknet-tools-cli starknet transfer-args ...` using these deno scripts -- but I wanted to avoid adding typescript code in our automated process.

Reply:
My thought was these values can constantly change (probably at least until the mainnet launch), so this part of the code could easily become outdated / confusing. Otherwise, we’d need to update these values every time we deploy a new set of contracts. Either way, I’ll leave that up to you. Maybe a short comment helps.

Thanks for adding these!

Reply:
on second thoughts -- it's a good idea. let's move these variables in `.env` file.

Reply:
6e6e702